source/server/admin/logs_handler.cc

```diff
@@ -52,13 +53,21 @@ Http::Code LogsHandler::handlerReopenLogs(absl::string_view, Http::ResponseHeade
 }
 
 bool LogsHandler::changeLogLevel(const Http::Utility::QueryParams& params) {
-  if (params.size() != 1) {
-    return false;
+  // When level is present, we only accept exact one param.
+  if (params.find("level") != params.end()) {
+    if (params.size() != 1) {
+      return false;
+    }
   }
 
-  std::string name = params.begin()->first;
-  std::string level = params.begin()->second;
+  bool ret = true;
+  for (const auto& [name, level] : params) {
```

Comment:
WDYT of having the name=level list be a single param so we could have
```
logging?level=<level>    changes levels of all paths
logging?paths=name1:value1,name2:value2,....

That would be a little nicer in that you could change the name of a path 'level', if that could ever be meaningful.

It would also fit a little bit better into a UI for the admin console I have in a pending PR https://github.com/envoyproxy/envoy/pull/19546

Reply:
@jmarantz Yes, I like the idea of combine name&level into a single param :) 
The latest commit accepts this form now: `logging?paths=name1:value1,name2:value2,...`